source/README.md

```diff
@@ -0,0 +1,77 @@
+# AERIUS File Server
```

Comment:
I would suggest to put the README.md in the root of the project so it will be shown when visiting the repo. Unless you intended a different readme for the ROOT.